cli/pcluster/utils.py

```diff
@@ -602,6 +602,23 @@ def get_master_ip_and_username(cluster_name):
     return master_ip, username
 
 
+def get_supported_archs_for_inst_type(instance_type):
+    """Get a list of architectures supported for the given instance type."""
+    try:
+        ec2_client = boto3.client("ec2")
```

Comment:
minor: you could avoid to define the ec2_client given that you're using it in the next row only.

`instance_info = boto3.client("ec2").describe_instance_types(InstanceTypes=[instance_type]).get("InstanceTypes")[0]`

Reply:
It was done this way in order to avoid line length linter errors or ugly formatting.